src/app/dataset/executions-data-grid/executions-data-grid.component.scss

```diff
@@ -3,6 +3,12 @@
 
 $width-orb: 3em;
 
+.buttonplaceholder {
+  &:empty {
+    display: none;
+  }
```

Comment:
If have seen this a couple of times in your code, could you say everything that is :empty should not be displayed? In other words, could this be a generic rule?

Reply:
In some contexts.  When the elements are stacked empty elements create irregular spacing.  When the elements are in the grid the empty cells have to be displayed for spacing.